source/extensions/filters/network/well_known_names.h

```diff
@@ -59,6 +59,8 @@ class NetworkFilterNameValues {
   const std::string ZooKeeperProxy = "envoy.filters.network.zookeeper_proxy";
   // WebAssembly filter
   const std::string Wasm = "envoy.filters.network.wasm";
+  // Copy dynamic metadata from downstream to upstream
+  const std::string DownToUp = "envoy.filters.network.down_up";
```

Comment:
This doesn't seem like the right place to specify a name as there is no filter. I think you want more something like shared metadata documented in https://www.envoyproxy.io/docs/envoy/latest/configuration/advanced/well_known_dynamic_metadata.

Reply:
So where should I put the metadata namespace definition ?

Reply:
I think something like "envoy.common.shared_network_metadata" might be appropriate? I am not good at naming, but most names are associated with filter names, and the PR is adding a common namespace. I think it's fine to keep it in this file.